src/DotNetWorker.Core/Converters/TypeConverter.cs

```diff
@@ -10,6 +10,7 @@ internal class TypeConverter : IInputConverter
     {
         public ValueTask<ConversionResult> ConvertAsync(ConverterContext context)
         {
+            // Option 1: If the target type is a string, check for null or empty on the source here
```

Comment:
I feel like this only solves the issue for strings and not all types/cases where we may want a null value

Reply:
Actually, it looks like source is always an empty string regarless of the binding type